src/System Application/App/Edit in Excel/src/Filters/EditInExcelFieldFilter.Interface.al

```diff
@@ -11,13 +12,23 @@ namespace System.Integration.Excel;
 interface "Edit in Excel Field Filter"
 {
     Access = Public;
+#if not CLEAN25
+    ObsoleteState = Pending;
+    ObsoleteTag = '25.0';
+#else
+    ObsoleteState = Removed;
+    ObsoleteTag = '28.0';
```

Comment:
Not sure we need this.

AFAIK the obsolete state removed is only for tables, because those persist in the SQL database even after obsoletion.

For interfaces, you should be able to just set the obsolete state to pending, tag to 25.0, and reason to your text. And instead of having the `#if` here, you can wrap the whole object in `#if not CLEAN25`, so that it goes away when we clean the 25 symbols.